lend/src/components/Common/ModalContent.tsx

```diff
@@ -236,3 +244,320 @@ const Wallet = () => {
 
   return content;
 };
+
+const LogoContent = styled.div`
+  width: 100%;
+  height: 160px;
+  padding: 40px 132px 40px 132px;
+  border-bottom: 1px solid ${colors.primaryText}1F;
+  background: linear-gradient(
+    102.28deg,
+    rgba(252, 10, 84, 0.04) 0%,
+    rgba(252, 10, 84, 0) 100%
+  );
+`;
+
+const LogoContent2 = styled.div`
```

Comment:
What are the differences between `LogoContent` and `LogoContent2`? Best if we could name them according to their differences